### ISSUE TITLE
Version 1.38.1 fails to install on Windows

### ISSUE THREAD

**Issue by westonpace** [jfrog/jfrog-cli#783]
**Describe the bug**
I run the command `npm install jfrog-cli-go@1.38.1` and get the following error:

```
C:\Users\westpace\temp>npm install jfrog-cli-go@1.38.1
npm WARN lifecycle The node binary used for scripts is C:\Program Files (x86)\Nodist\bin\node.exe but npm is using C:\Program Files (x86)\Nodist\v-x64\10.16.0\node.exe itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> jfrog-cli-go@1.38.1 preinstall C:\Users\westpace\temp\node_modules\jfrog-cli-go
> node init.js

Downloading JFrog CLI 1.38.1
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open 'C:\Users\westpace\temp\node_modules\jfrog-cli-go\bin\jfrog.exe'
```

**To Reproduce**
This has happened on multiple Windows servers.

**Expected behavior**
A successful install.  I have no issues with 1.38.0

**Screenshots**

**Versions**
- JFrog CLI version: 1.38.1
- JFrog CLI operating system: Windows 10 Build 
- Artifactory Version: N/A

**Additional context**
I first noticed this on a Jenkins server which has been running daily without issue for months so I feel pretty sure it is related to the 1.38.1 release.

**Comment by eyalbe4** [comment]
Thanks for reporting this issue @westonpace!
We found the root cause of this issue. It appears that it is the result of a restructure made to the build pipeline.
We're working on releasing a fix and we'll let you know as soon as it is out.
Please accept my apologies for the inconvenience caused by this.

**Comment by eyalbe4** [comment]
@westonpace,
We released version 1.38.2, which resolves the issue.
Looking forward to your feedback for it.

**Comment by westonpace** [comment]
Thank you for the quick turnaround.  1.38.2 works for me.
